nucypher/cli/commands/stake.py

```diff
@@ -948,3 +949,54 @@ def set_min_rate(general_config, transacting_staker_options, config_file, force,
                           receipt=receipt,
                           chain_name=blockchain.client.chain_name,
                           transaction_type='set_min_rate')
+
+
+@stake.command('create-workers')
+@group_staker_options
+@option_config_file
+@click.option('--cloudprovider', help="currently aws", default='aws')
+@click.option('--aws-profile', help="The AWS account profile you'd like to use", default='default')
+@click.option('--remote-provider', help="the blockchain provider for the remote node")
+@click.option('--clef', help="use clef for remote signing", default=False)
+@group_general_config
+def create_workers(general_config, staker_options, config_file, cloudprovider, aws_profile, remote_provider, clef):
+    """Creates workers for all stakes owned by the user for the given network."""
+
+    emitter = setup_emitter(general_config)
+
+    STAKEHOLDER = staker_options.create_character(emitter, config_file)
+
+    stakers = STAKEHOLDER.get_stakers()
+    if not stakers:
+        emitter.echo("No staking accounts found.")
+
+    stakes_to_deploy = []
+    for staker in stakers:
```

Comment:
@KPrasch check this part out.  Does this make sense?

Reply:
This is actually what prompted my comment, It appears as if you are provisioning one host per stake, which is too many.  Instead, there needs to be one worker-per-staker.